.local/bin/channelrefresh

```diff
@@ -15,7 +16,16 @@ error_handling() {
 	    notify-send "'channels.txt' formatting is wrong."
 	    exit 1
     }
-}
+    for pkg in yt-dlp mpv jq; do
+        command -v "$pkg" >/dev/null || {
+            notify-send "$pkg is not installed. Installing..."
+            $INSTALLER "$pkg" || {
```

Comment:
Since the installer variable is used in only 1 place, it shouldn't be a variable imo. It's also kinda spooky for a new user to open a random bin script and see the first variable be INSTALLER xD